app/utils/drivers.js

```diff
@@ -15,10 +15,11 @@
  * == BSD2 LICENSE ==
  */
 
-/* global __REDUX_DEV_UI__ */
+import os from 'os';
 
-if (__REDUX_DEV_UI__ === true) {
-  module.exports = require('./Root.dev');
+const platform = os.platform();
+if (platform === 'darwin'){
```

Comment:
The drivers detection is split like this because the two code paths depend on node modules which will only be installed/available on a particular OS and will error out if you try to require them on a non-supported OS.